benchmarks/benchmark_coo.py

```diff
@@ -33,3 +38,24 @@ def time_add(self):
 
     def time_mul(self):
         self.x * self.y
+
+
+class IndexingSuite(object):
+    def setup(self):
+        np.random.seed(0)
+        self.x = sparse.random((100, 100, 100), density=0.01)
+        self.x.sum_duplicates()
+
+        self.x[5]  # Numba compilation
+
+    def time_index_scalar(self):
+        self.x[5]
+
+    def time_index_slice(self):
+        self.x[:50]
+
+    def time_index_slice2(self):
+        self.x[:50, :50]
```

Comment:
Are different steps likely to make a difference here?

Reply:
What matters most here is the length of the slice and the number of nonzeros within it.